### PR TITLE
rewrite ldbc snb queries

### PR DESCRIPTION
**Pull request by anuchak**

### CODE REVIEW

**Comment by anuchak** [comment]
#1204 - the results are now visible and the tests are passing: https://kuzu-benchmark.mew.im/run.html#831

**Comment by mewim** [comment]
Only Q32, Q34, and Q37 got logged into the benchmark results. Apparently Q36 and Q38 run out of memory. I would suggest to remove Q36 and Q38 and try again.

**Comment by mewim** [comment]
It seems that we actually have another issue in the benchmark runner: https://github.com/kuzudb/kuzu/pull/1234. Maybe try to run the benchmarks again after this PR is merged.

File: benchmark/queries/ldbc-sf100/ldbc_snb_ic/q35.benchmark [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-NAME q37
```
Comment (mewim) [inline benchmark/queries/ldbc-sf100/ldbc_snb_ic/q35.benchmark:1]: Change to Q35

File: benchmark/queries/ldbc-sf100/ldbc_snb_ic/q36.benchmark [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-NAME q40
```
Comment (mewim) [inline benchmark/queries/ldbc-sf100/ldbc_snb_ic/q36.benchmark:1]: Change to Q36

**Review by mewim — APPROVED** [review]
I can see the queries passing on Benchmark Result Overview. Just change the two query's name to be consistent with the file name and I think we are good to merge.

File: benchmark/queries/ldbc-sf100/ldbc_snb_ic/q35.benchmark (hunk 1 above)
Comment (anuchak) [reply benchmark/queries/ldbc-sf100/ldbc_snb_ic/q35.benchmark]: done

File: benchmark/queries/ldbc-sf100/ldbc_snb_ic/q36.benchmark (hunk 2 above)
Comment (anuchak) [reply benchmark/queries/ldbc-sf100/ldbc_snb_ic/q36.benchmark]: done

**Pull request merged**